shared/src/common/types/user/user-sign-up-response-dto.type.ts

```diff
@@ -1,8 +1,10 @@
+import { PermissionsGetAllItemResponseDto } from '../permission/permission-item-response-dto.type';
 import { UsersByIdResponseDto } from './users-by-id-response-dto.type';
 
 type UserSignUpResponseDto = {
   token: string;
   user: UsersByIdResponseDto;
+  permissions: PermissionsGetAllItemResponseDto[];
```

Comment:
```suggestion
  permissions: PermissionsGetAllItemResponseDto[];
```
should be inside the user object